assets/css/index.css

```diff
@@ -80,3 +80,11 @@ hr {
   font-family: 'Archivo Narrow', sans-serif;
   font-size: 120px;
 }
+
+.testimonials__faces {
+    height: 10vh;
```

Comment:
I'll be interested to see how this responds in mobile devices or large screens. There are no vmax or vmin specified so I assume that when the screen is small the images will be _really_ small.
It's an area I'm interested in looking into https://css-tricks.com/css-locks/